shared/constants.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         node: {
             checkCmd: 'node --version',
             minVersion: '6.9',
-            maxVersion: '10.11',
+            maxVersion: '10.11'
```

Comment:
Unrelated to this PR, but just saw that 11.1 is out, I'll test it and update this value if there is no issues.